skel/daemons/license.go

```diff
@@ -56,7 +56,7 @@ func LicenseRenewal(appName string, app *sonic.Runtime, vaultService *vault.Serv
 		for {
 			lic, err := getLicenseFromVault(appName, vaultService)
 			if err != nil {
-				log.Printf("ERR: LicenseRenewal: %s", err)
```

Comment:
There's no need to rename ERR to ERROR, let's save some space in the logs

Reply:
And just to clarify, ERR is just a prefix we use to distinguish errors from everything else